taxonomies/food/ingredients.txt

```diff
@@ -52515,6 +52561,11 @@ carbon_footprint_fr_foodges_value:fr: 1.2
 ciqual_food_code:en: 20026
 ciqual_food_name:en: Chicory, raw
 ciqual_food_name:fr: Endive, crue
+ecobalyse:en: endive-eu
+ecobalyse_labels_en_organic:en: endive-eu
+ecobalyse_origins_en_france:en: endive-fr
+ecobalyse_origins_en_european_union:en: endive-eu
```

Comment:
no need for ecobalyse_labels_en_organic:en: endive-eu and ecobalyse_labels_en_organic_origins_en_france:en: endive-fr